src/helpers/vars.ts

```diff
@@ -0,0 +1,3 @@
+const DELAY_OF_FILTERING = 1000;
+
+export { DELAY_OF_FILTERING };
```

Comment:
`vars` stands for variables :)

In your case `constants` is probably a better file name